### PR TITLE
pull request

### PR DESCRIPTION
**Pull request by RoberMac**

### CODE REVIEW

**Comment by RoberMac** [comment]
@jimaek

File: files/podpicker/update.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
   "name": "PodPicker",
   "repo": "RoberMac/PodPicker",
   "files": {
-    "basePath": "dist/",
-    "exclude": ["PodPicker.min.js", "PodPicker.min.css"]
```
Comment (jimaek) [inline files/podpicker/update.json]: replace "exclude" with "include". We only need the minified files, not everything

Comment (RoberMac) [inline files/podpicker/update.json]: @jimaek OK👌

**Comment by RoberMac** [comment]
@jimaek

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**